frontend/wallet/schema.graphql

```diff
@@ -266,14 +268,14 @@ type Subscription {
   # Subscribe to sync status of the node
   newSyncUpdate: SyncUpdate!
   
-  # Subscribe to payments for which this key is the sender or receiver
-  newUserCommandUpdate(filterBySenderOrReceiver: String!): UserCommandUpdate!
+  blockConfirmation: BlockConfirmation!
   
-  # Subscribe all blocks created by `key`
-  newBlock(key: String): BlockUpdate!
+  # Subscribe to all blocks related to `key`
+  newBlock(key: String): Block!
   
-  # Subscribe to fees earned by key
-  newSnarkFee(key: String): SnarkFee!
+  # Triggered whenever the best tip changes
+  # to another branch in the transition frontier
+  reorg: Boolean!
```

Comment:
I would not use abbreviations. Just name it `reorganize`.

When a reorg happens, we can get the added blocks that got introduced into the longest chain and the removed blocks from the longest chain.

Would it be better to return this type of information rather than a bool?

Reply:
for now we said we'd just refresh, meaning there is no meaningful data associated with the event. Later we could make this return some information about the change I think, yeah.
Will change to "reorganizeEvent" or something

Reply:
okay sounds good to me.